src/pages/workspaces/workspace/jobHistory/CallTable.js

```diff
@@ -100,7 +100,7 @@ const doesTaskHaveCostData = (task) => {
 };
 
 const noCostData = (task, subWorkflowId) => {
-  if (task?.executionStatus === 'Failed' || task?.callCaching?.hit === true || !_.isEmpty(subWorkflowId) || !task.startTime) {
+  if (task?.executionStatus === 'Failed' || task?.callCaching?.hit === true || !_.isEmpty(subWorkflowId) || !task?.startTime) {
```

Comment:
TOL: what it help to have unit test for this?

Reply:
+1 I think it should

Reply:
@kpierre13 do you think it would be possible to add a table-based testing or parameterized test for this specific method noCostData? I think apart from the test you have already added it would be nice to have a unit test for this method too